src/TranslationUi/Term/Field/Title.php

```diff
@@ -0,0 +1,79 @@
+<?php # -*- coding: utf-8 -*-
+/*
+ * This file is part of the MultilingualPress Extensions Boilerplate package.
+ *
+ * (c) Inpsyde GmbH
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Inpsyde\MultilingualPress\YoastSeoSync\TranslationUi\Term\Field;
+
+use Inpsyde\MultilingualPress\TranslationUi\MetaboxFieldsHelper;
+use Inpsyde\MultilingualPress\TranslationUi\Term\RelationshipContext;
+use Inpsyde\MultilingualPress\YoastSeoSync\TranslationUi\Term\MetaboxFields;
+
+class Title
+{
+    /**
+     * @param MetaboxFieldsHelper $helper
+     * @param RelationshipContext $context
+     */
+    public function __invoke(MetaboxFieldsHelper $helper, RelationshipContext $context)
+    {
+        $id = $helper->fieldId(MetaboxFields::FIELD_TITLE);
+        $name = $helper->fieldName(MetaboxFields::FIELD_TITLE);
+        $value = $this->value($context);
+
+        ?>
+        <tr>
+            <th scope="row">
+                <label>
+                    <?php esc_html_e('SEO title', 'multilingualpress-yoast-seo-sync') ?>
+                </label>
+            </th>
+            <td>
+                <input type="text"
+                       class="large-text"
+                       name="<?= esc_attr($name) ?>"
+                       id="<?= esc_attr($id) ?>"
+                       value="<?= esc_attr($value) ?>"/>
```

Comment:
@Dinamiko This will not allow html, what yoast do in this case? If html isn't allowed ok but I would use `esc_html` instead. If html is allowed I would switch to the same escape function used by Yoast.